src/index.html

```diff
@@ -166,63 +166,150 @@ <h1>Scaling <i>beyond</i> geospatial tech</h1>
-          <div class="sm:grid sm:grid-cols-3">
-            <p class="tech-spec text-white">
-              ONE Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="tech-spec text-white">
-              TWO Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="tech-spec text-white">
-              THREE Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="tech-spec text-white">
-              FOUR Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="tech-spec text-white">
-              FIVE Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="tech-spec text-white">
-              SIX Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="tech-spec text-white">
-              SEVEN Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="tech-spec text-white">
-              EIGHT Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="tech-spec text-white">
-              NINE Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="tech-spec text-white">
-              TEN Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
+          <div class="flex justify-center py-4">
+            <button
+              id="spec-button"
+              class="sm:hidden text-white border-gray-button border-opacity-10 border-1 py-2 px-10 font-semibold"
+            >
+              See more specs
+            </button>
           </div>
-          <div class="text-center">
+          <div id="spec-section" class="hidden sm:block sm:mb-28">
+            <h1 class="text-center text-white">Technical specifications</h1>
+            <div class="sm:grid sm:grid-cols-2 md:grid-cols-3">
+              <div class="tech-spec text-white">
+                <img src="images/timeline-arrow.svg" class="h-7 w-7 pb-2" />
```

Comment:
Now that we have 3 classes in our icon class list, I think it's worth moving these into a new `tech-spec-icon` class to make any future updates on these easier.